pyseir/icu/infer_icu.py

```diff
@@ -64,6 +100,36 @@ def get_icu_timeseries(
     )
 
 
+def get_icu_timeseries_from_regional_input(
+    regional_input: RegionalInput,
+    use_actuals: bool = True,
+    weight_by: ICUWeightsPath = ICUWeightsPath.POPULATION,
+) -> OneRegionTimeseriesDataset:
+    """
+    Loads data for region of interest and returns ICU Utilization numerator.
+
+    Args:
+        region: Region of interest
+        use_actuals: If True, return actuals when available. If False, always use predictions.
+        weight_by: The method by which to estimate county level utilization from state level utilization when
+            no county level inpatient/icu data is available.
+
+    Returns:
+        A date-indexed series of ICU estimate for heads-in-beds for a given region
```

Comment:
Remove "A date-indexed series of " because the details of what indexes are present is meant to be encapsulated within OneRegionTimeseriesDataset